vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/view/components/ProjectDetailsComponent.java

```diff
@@ -219,18 +227,14 @@ public Handler(ProjectInformationService projectInformationService,
       setupExperimentalDesignSearch();
     }
 
-    public void projectId(String projectId) {
-      try {
-        parseProjectId(projectId);
-        projectInformationService.find(ProjectId.parse(projectId))
-            .ifPresentOrElse(
-                this::loadProjectData,
-                () -> titleToggleComponent.setValue("Not found"));
-      } catch (AccessDeniedException accessDeniedException) {
-        log.error("Access denied when loading project details for project id " + projectId,
-            accessDeniedException);
-        titleToggleComponent.setValue("Not found");
-      }
+    public void setProjectId(String projectId) {
+      parseProjectId(projectId);
+      projectInformationService.find(projectId).ifPresentOrElse(this::loadProjectData,
+          this::emptyAction);
+    }
+
+    //ToDo what should be done if projectID could not be retrieved
+    private void emptyAction() {
```

Comment:
should not stay like this.
I would say show error message that the project could not be found